assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -24,6 +25,16 @@ import {
   sapInstancesTableConfiguration,
 } from './tableConfigs';
 
+const semver = require('semver');
+
+export const agentVersionWarnings = (agentVersion) => {
```

Comment:
I'd name this just `agentVersionWarning` and put it in a dedicated library instead of just exporting it from `HostDetails`

Reply:
Okey, not strong opinions of this. I thought that it would be an overkill to do that, but I will include a new file in the `lib` folder with this functionalities.